auto_nag/scripts/good_first_bug_unassign_inactive.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+class GoodFirstBugUnassignInactive(BzCleaner):
+    def __init__(self):
+        super(GoodFirstBugUnassignInactive, self).__init__()
+        self.default_assignee = utils.get_config(self.name(), "default_assignee")
+        self.nmonths = utils.get_config(self.name(), "months_lookup")
+
+        self.autofix_assignee = {}
+
+    def description(self):
+        return "Bugs with good-first-bug keyword and no activity for the last {} months".format(
+            self.nmonths
+        )
+
+    def get_autofix_change(self):
+        return self.autofix_assignee
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        doc = self.get_documentation()
+
+        self.autofix_assignee[bugid] = {
+            "comment": {
+                "body": "This good-first-bug hasn't had any activity for {} months, it is automatically unassigned.\n{}".format(
+                    self.nmonths, doc
```

Comment:
Since you added doc, don't forget to create it here: https://wiki.mozilla.org/Release_Management/autonag.

Reply:
Done.